CONTRIBUTING.md

```diff
@@ -163,3 +173,180 @@ describe 'a dog', ->
+
+#### Type of Issue and Issue State
+
+| Label name | `atom/atom` :mag_right: | `atom`‑org :mag_right: | Description |
+| --- | --- | --- | --- |
+| `enhancement` | [search][search-atom-repo-label-enhancement] | [search][search-atom-org-label-enhancement] | A feature request. |
+| `bug` | [search][search-atom-repo-label-bug] | [search][search-atom-org-label-bug] | A confirmed bug or issue very likely to be a bug. |
+| `question` | [search][search-atom-repo-label-question] | [search][search-atom-org-label-question] | A question more than a bug report or feature request (e.g. how do I do X). |
+| `feedback` | [search][search-atom-repo-label-feedback] | [search][search-atom-org-label-feedback] | General feedback more than a bug report or feature request. |
+| `help-wanted` | [search][search-atom-repo-label-help-wanted] | [search][search-atom-org-label-help-wanted] | The Atom core team would appreciate help from the community in resolving these issues. |
+| `beginner` | [search][search-atom-repo-label-beginner] | [search][search-atom-org-label-beginner] | Less complex issue which would be a good first issue to work on for users who want to contribute to Atom. |
+| `more-information-needed` | [search][search-atom-repo-label-more-information-needed] | [search][search-atom-org-label-more-information-needed] | More information needs to be collected about the problem or feature request (e.g. steps to reproduce). |
+| `needs-reproduction` | [search][search-atom-repo-label-needs-reproduction] | [search][search-atom-org-label-needs-reproduction] | Likely a bug, but hasn't been reliably reproduced. |
+| `triage-help-needed` | [search][search-atom-repo-label-triage-help-needed] | [search][search-atom-org-label-triage-help-needed] | Help is needed to triage this issue, normally from the Atom core team, e.g. to determine if it's a bug or not. |
+| `blocked` | [search][search-atom-repo-label-blocked] | [search][search-atom-org-label-blocked] | Issue is blocked on another issue. |
+| `duplicate` | [search][search-atom-repo-label-duplicate] | [search][search-atom-org-label-duplicate] | Issue is a duplicate of another issue. |
+| `wontfix` | [search][search-atom-repo-label-wontfix] | [search][search-atom-org-label-wontfix] | The Atom core team decided not to fix this issue for now, either because it's working as intended or for some other reason. |
+| `invalid` | [search][search-atom-repo-label-invalid] | [search][search-atom-org-label-invalid] | Issue isn't valid (e.g. user error). |
+| `package-idea` | [search][search-atom-repo-label-package-idea] | [search][search-atom-org-label-package-idea] | A feature request which might be a good candidate for a new package, instead of extending Atom or a core Atom package. |
+| `wrong-repo` | [search][search-atom-repo-label-wrong-repo] | [search][search-atom-org-label-wrong-repo] | Issue was reported on the wrong repository (e.g. a bug related to a package was reported on `atom/atom`). |
```

Comment:
It might be better to be more specific here, eg `a bug related to the settings view was reported on atom/atom`.